### DISCUSSION TITLE
Possible to set new baseline on existing Prisma 3 project?

### DISCUSSION THREAD

**Discussion by wiput1999** [prisma/prisma#12763] [Q&A] [answered]
Hi,
   I'm using Prisma 3 on production now I've quite a lot of migrations folders like hundreds of migrations. Does Prisma have the best practice to do like setup a new baseline from hundreds of migration into like a new initial migration?

DB : Postgres

Example
Old -> 400 Migrations
New Baseline -> 1 Migration with script that create all existing models

Or we shouldn't do that should keep it incrementing over time?

**Comment by nurul3101** [comment, marked as answer]
Hey @wiput1999 👋 

It seems that you are basically describing the squashing of migration files.
We are working on documenting how you could achieve squashing with the new `prisma migrate diff` [commands](https://www.prisma.io/docs/reference/api-reference/command-reference#migrate-diff).

Here's the GitHub Issue for squashing migrations: [#5714](https://github.com/prisma/prisma/issues/5714).
You can try out the workflow mentioned in this [comment](https://github.com/prisma/prisma/issues/5714#issuecomment-1043163364) to get one squashed migration file.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
